go-fips-1.21.yaml

```
@@ -0,0 +1,104 @@
+package:
+  name: go-fips-1.21
+  version: 1.21.1
+  epoch: 0
+  description: "the Go programming language with OpenSSL cryptography"
+  copyright:
+    - license: BSD-3-Clause
+  dependencies:
+    provides:
+      - go-fips=1.21.999 # This should be go-fips=${{package.version}}-${{package.epoch}}
```

```suggestion
      - go-fips=${{package.version}}-${{package.epoch}}
```

I think this works now: https://github.com/chainguard-dev/melange/pull/610